.npmrc

```diff
@@ -1,2 +1 @@
-audit-level = high
-audit = false
+save-exact = true
```

Comment:
how come we've removed ```audit-level = high``` ?

Reply:
resolving this - Jon has now double-checked current config runs the proper auditing process in the pipeline :)